src/bitfield.rs

```diff
@@ -26,7 +26,7 @@ pub trait Facter<T: From<u32> + From<bool>> {
 }
 
 ///Wraps a bit flag, usually representing if a feature is present or not
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug, Clone)]
```

Comment:
at what point do these fields get cloned in this PR?

Reply:
https://github.com/ScaleComputing/cpuinfo/pull/19#discussion_r1735011031